third_party/wabt/src/interp/jit/jit-backend.cc

```diff
@@ -30,7 +28,7 @@
 #endif
 
 extern "C" {
-#include "../../../../sljit/sljitLir.c"
+#include "sljitLir.c"
```

Comment:
The point is not including the file anymore. It should be compiled as a `.o` file and linked.